ЦелочисленнаяАрифметика/ЦелочисленнаяАрифметика/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ЦелочисленнаяАрифметика
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите двухзначное число:");
+
+            int b;
+
+            b = int.Parse(Console.ReadLine());
+            Console.WriteLine("десятки числа - " + (b / 10).ToString());
+
+            Console.WriteLine("единицы числа - " + (b % 10).ToString());
+
+            int S;
+
+            S = ((b / 10) + (b % 10));
+
+            Console.WriteLine("Сумма - " + S.ToString());
+
+            int P;
+
+            P = ((b / 10) * (b % 10));
+            Console.WriteLine("Произведение - " + P.ToString());
+
+
+
+
+
+            Console.ReadKey();
```

Comment:
Все верно, но величины `b / 10`  и `b % 10` вы вычисляете по нескольку раз. Лучше вычислить один раз и запомнить эти значения в локальных переменных. Локальные переменные в C# принято назвать с маленькой буквы. И нет злоупотребляйте пустыми строками. Используйте их для визуального разделения кода на смысловые части. Если приписываете к строке число, метод ToString() вызывать не обязательно. 
```suggestion
            Console.WriteLine("Введите двухзначное число:");
            int b = int.Parse(Console.ReadLine());
            
            var units = b % 10;
            var tens = b / 10;
            
            Console.WriteLine("десятки числа - " + tens);
            Console.WriteLine("единицы числа - " + units);
            Console.WriteLine("Сумма цифр - " + (units + tens));
            Console.WriteLine("Произведение - " + units * tens);

            Console.ReadKey();
```
Сумму нужно взять в скобки, иначе оба числа припишутся к строке, а не просуммируются.
Произведение брать в скобки не обязательно, операция * имеет приоритет перед операцией +.